addon/components/detail.js

```diff
@@ -101,6 +101,7 @@ export default Component.extend(PropTypeMixin, {
     ]),
     hook: PropTypes.string,
     onChange: PropTypes.func,
+    tabName: PropTypes.string,
```

Comment:
Also please put the property in the correct location in the `propTypes` object to maintain alphabetical sorting.